manticore/platforms/evm.py

```diff
@@ -32,6 +32,7 @@
 MASK160 = 2 ** 160 - 1
 TT255 = 2 ** 255
 TOOHIGHMEM = 0x1000
+DEFAULT_FORK = 'byzantium'
```

Comment:
Should probably be a config variable/parameter, but we can make it after merging.

Reply:
So far this is only used to tune the unittests that needs older-than-byzantium assembler. Once we pass frontier in both concrete AND symbolic we are probably deleting those tests and upgrade to the next fork. So in the end we'll end up not needing this variable/constant/config thing.